vm_trace.c

```diff
@@ -937,6 +937,9 @@ rb_tracearg_parameters(rb_trace_arg_t *trace_arg)
             const rb_method_entry_t *me;
             VALUE iclass = Qnil;
             me = rb_method_entry_without_refinements(trace_arg->klass, trace_arg->called_id, &iclass);
+            if (!me) {
+              me = rb_method_entry_without_refinements(trace_arg->klass, trace_arg->id, &iclass);
```

Comment:
The indentation used in the C code is 4 spaces:

```suggestion
                me = rb_method_entry_without_refinements(trace_arg->klass, trace_arg->id, &iclass);
```

Reply:
Done. Thanks!